helpers/filterSeats.js

```diff
@@ -21,8 +24,11 @@ module.exports = function(dust) {
     var venueProduct = _.head(params.venueProducts);
     var ticketRates = params.ticketRates;
     var roomRates = params.roomRates;
-    var bandColours = params.bandColours;
-    var sortByInput = params.sortBy;
+    var sortType = params.sortBy;
```

Comment:
@putvande bootstrap gives us the CSS for free with text-capitalize, I'm using it on theatreBlueprint.

@owennicol  Yes it should have been like you said, my bad, will update it in a minute